tekton/ci/stepactions/gcs-upload.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: tekton.dev/v1alpha1
+kind: StepAction
+metadata:
+  name: gcs-upload-sa
+  labels:
+    app.kubernetes.io/version: "0.3"
```

Comment:
Agreed. I removed this label since it is a local step action for now.

Reply:
I can take a stab at what it would take to host it in the catalog. I believe when I reached out originally, a TEP was required which will take some effort. I thought that in the meantime, I'd add these here just so that we can dogfood something. As soon as we have a catalog for StepActions, I can remove these and instead use the catalog step actions.